source/js/components/pulse-project-list/pulse-project-list.jsx

```diff
@@ -69,7 +69,9 @@ export default class PulseProjectList extends React.Component {
       let url;
 
       if (project.related_creators.length) {
-        byline = `By ${project.related_creators.map(rc => rc.name).join(`, `)}`;
+        byline =
+          gettext("By") +
```

Comment:
This makes no sense, template literals are explicitly to _avoid_ string concatenation =)
```
          byline = `${gettext("By")} ${project. ..... }`;
```